plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -335,15 +334,6 @@ def deploy_to_cluster(release, release_docs)
       successful
     end
 
-    # Create the service or report it's status
-    def ensure_service(release_docs)
```

Comment:
atm we were playing favorites ... but I think we should not touch the template ordering ...